src/components/modules/VehicleList/index.jsx

```diff
@@ -70,4 +32,10 @@ class VehicleList extends React.Component {
   }
 }
 
-export default VehicleList;
+VehicleList.propTypes = {
+  match: PT.shape({
+    params: PT.object,
+  }),
```

Comment:
As `match` comes from `withRouter` and is always available, you can make it `isRequired`.